src/modules/Dropdown/DropdownMenu.js

```diff
@@ -8,13 +8,16 @@ import {
   META,
 } from '../../lib'
 
-function DropdownMenu(props) {
-  const { children, className } = props
-  const classes = cx('menu transition', className)
-  const rest = getUnhandledProps(DropdownMenu, props)
-  const ElementType = getElementType(DropdownMenu, props)
+export default class DropdownMenu extends Component {
+  render() {
+    const { children, className } = this.props
+    const classes = cx('menu transition', className, { visible: this.props.opened })
```

Comment:
Please update this className buildup to be consistent with other components:

- destructure props
- utilize `useKeyOnly` for the `visible` class
- put the user's `className` last in the list